### ISSUE TITLE
Screen washed out when enabling HDR

### ISSUE THREAD

**Issue by aberthil** [itsmikethetech/Virtual-Display-Driver#94]
As soon as enabled HDR, the screen is washed out, tried to run HDR calibration but does not fix the issue, any suggestion. This happens regardless of the resolution.....
![Screenshot 2024-05-25 at 5 20 55 PM](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/75104089/90d6e0f2-fb71-47a1-9706-b19ab3f7836e)
![Screenshot 2024-05-25 at 5 21 09 PM](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/75104089/d5a5db61-67cb-4197-91a3-8b48f7b19e11)

**Comment by aberthil** [comment]
Even more clear using qbitorrent
![Screenshot 2024-05-25 at 5 32 31 PM](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/75104089/05465f41-ae8b-4296-ab2a-55bd68dcddcc)
![Screenshot 2024-05-25 at 5 32 45 PM](https://github.com/itsmikethetech/Virtual-Display-Driver/assets/75104089/d81f53fc-73e8-4fc5-8c4f-c922e2763d3c)

**Comment by aberthil** [comment]
One more data point, it happens only when displaying the desktop, the games are working great with HDR enabled, seems like something missing somewhere....

**Comment by A-m-a-a-n** [comment]
I have this issue as well.

**Comment by BearTS** [comment]
what client are you using to stream? 
I had this issue why my gpu didnt support the encoding of HDR, got it fixed by forcing it another gpu (its a laptop so yeah 2 gpu)

**Comment by aberthil** [comment]
Moonlight is the clientOn Jul 2, 2024, at 12:52 PM, Anuj Parihar ***@***.***> wrote:﻿
what client are you using to stream?
I had this issue why my gpu didnt support the encoding of HDR, got it fixed by forcing it another gpu (its a laptop so yeah 2 gpu)

**Comment by BearTS** [comment]
Did you turn on HDR streaming (Experimental) in settings?

**Comment by aberthil** [comment]
Yes it is enabled.
Alexis

**Comment by BearTS** [comment]
in that case, make sure your pc support streaming HDR 
and also make sure your client has HDR enabled

**Comment by aberthil** [comment]
Yep this is the case for bothOn Jul 6, 2024, at 8:59 PM, Anuj Parihar ***@***.***> wrote:﻿
in that case, make sure your pc support streaming HDR
and also make sure your client has HDR enabled

**Comment by BearTS** [comment]
are you able to enable HDR in display settings in Windows? 
If that is the case, you should ask sunshine/moonlight folks, not here.

**Comment by matheusfaustino** [comment]
Could be related to this https://github.com/itsmikethetech/Virtual-Display-Driver/issues/116#issuecomment-2261242983 . It was happening for me as well, and then I calibrated the HDR and did this workaround and now it is fine
